pkgs/applications/networking/ipfs/default.nix

```diff
@@ -1,21 +1,21 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   name = "ipfs-${version}";
```

Comment:
buildGoModule won't accept `pname =` ?

Reply:
it does.

Reply:
Yes it does, I just missed it :innocent: .  Updated accordingly, thanks!